app/presenters/content_workflow_presenter.rb

```diff
@@ -0,0 +1,35 @@
+class ContentWorkflowPresenter < CSVPresenter
+  def initialize(scope = Edition.not_in(state: ["archived"]).all)
+    super(scope)
```

Comment:
You don't technically need to pass `scope` in here as the default behaviour for `super` will pass all method arguments to the superclass.